react/src/pages/utils/typings.tsx

```diff
@@ -167,3 +167,5 @@ export interface Field {
     fieldName?: string;
     disableEdit?: boolean;
 }
+
+export type StringTable = Array<Array<string>>;
```

Comment:
You can just do `string[][]` where necessary. In general you can do `type[]` instead of `Array<type>`.